### PR TITLE
[4.3] re apply 4.3-dev #2691

### PR DESCRIPTION
**Pull request by zero-24**
re apply the changes from https://github.com/joomlagerman/joomla/pull/2691#issue-1501902367

### CODE REVIEW

File: administrator/language/de-DE/com_templates.ini [hunk 1]
```diff
@@ -262,4 +262,4 @@ COM_TEMPLATES_WARNING_FORMAT_WILL_NOT_BE_VISIBLE="Es wurde eine neue Datei mit d
...
+JLIB_RULES_SETTING_NOTES_COM_TEMPLATES="Vorgenommene Änderungen wirken sich nur auf diese Komponente aus.<br /><em><strong>Vererbt</strong></em>: Eine globale Konfigurationseinstellung oder eine höhere Gruppeneinstellung wird angewendet.<br /><em><strong>Verweigert</strong></em>: Wird immer angewendet, egal was als globale Konfigurationseinstellung oder in einer höheren Gruppeneinstellung definiert wurde. Vererbt sich auf alle Unterelemente.<br /><em><strong>Erlaubt</strong></em>: Aktiviert die Aktion für diese Komponente, sofern es nicht durch eine globale Konfigurationseinstellung überschrieben wird.<br />Der Eintrag muss gespeichert werden, damit sich die errechneten Einstellungen aktualisieren können."
```
Comment (heelc29) [inline administrator/language/de-DE/com_templates.ini]: blank line at eof

File: administrator/language/de-DE/mod_quickicon.ini [hunk 2]
```diff
@@ -27,35 +36,50 @@ MOD_QUICKICON_MENUITEMS_MANAGER_ADD="Neuer Menüeintrag"
...
+MOD_QUICKICON_TEMPLATE_STYLES="Template Stile"
 MOD_QUICKICON_TYPE="Symboltyp"
 MOD_QUICKICON_UPDATE="Update"
 MOD_QUICKICON_USER_MANAGER="Benutzer"
 MOD_QUICKICON_USER_MANAGER_ADD="Neuer Benutzer"
 MOD_QUICKICON_WITH_COUNT="Symbole mit Anzahl anzeigen"
-MOD_QUICKICON_XML_DESCRIPTION="Dieses Modul zeigt Schnellsymbole, die auf dem Dashboard (Startseite im Administratorbereich) sichtbar sind."
+MOD_QUICKICON_WORKFLOW_MANAGER="Workflows"
+MOD_QUICKICON_WORKFLOW_MANAGER_ADD="Neuer Workflow"
+MOD_QUICKICON_XML_DESCRIPTION="Dieses Modul zeigt Schnellsymbole, die auf dem Dashboard (Startseite im Administratorbereich) sichtbar sind."
```
Comment (heelc29) [inline administrator/language/de-DE/mod_quickicon.ini]: blank line at eof

File: administrator/language/de-DE/com_templates.ini (hunk 1 above)
Comment (zero-24) [reply administrator/language/de-DE/com_templates.ini]:
```suggestion
JLIB_RULES_SETTING_NOTES_COM_TEMPLATES="Vorgenommene Änderungen wirken sich nur auf diese Komponente aus.<br /><em><strong>Vererbt</strong></em>: Eine globale Konfigurationseinstellung oder eine höhere Gruppeneinstellung wird angewendet.<br /><em><strong>Verweigert</strong></em>: Wird immer angewendet, egal was als globale Konfigurationseinstellung oder in einer höheren Gruppeneinstellung definiert wurde. Vererbt sich auf alle Unterelemente.<br /><em><strong>Erlaubt</strong></em>: Aktiviert die Aktion für diese Komponente, sofern es nicht durch eine globale Konfigurationseinstellung überschrieben wird.<br />Der Eintrag muss gespeichert werden, damit sich die errechneten Einstellungen aktualisieren können."

```

File: administrator/language/de-DE/mod_quickicon.ini (hunk 2 above)
Comment (zero-24) [reply administrator/language/de-DE/mod_quickicon.ini]:
```suggestion
MOD_QUICKICON_XML_DESCRIPTION="Dieses Modul zeigt Schnellsymbole, die auf dem Dashboard (Startseite im Administratorbereich) sichtbar sind."

```

**Comment by zero-24** [comment]
Fixed thanks @heelc29

**Comment by heelc29** [comment]
Is it possible that this PR #2616 is missing?

**Comment by zero-24** [comment]
Yes applied via https://github.com/joomlagerman/joomla/pull/2694/commits/a71fbdcb9064b8823a1c929af1e4ac38f8d377fb

**Review by tecpromotion — DISMISSED** [review]
Thanks @zero-24 
@heelc29 ready to merge?

**Review by heelc29 — DISMISSED** [review]
LGTM

File: administrator/language/de-DE/com_workflow.ini [hunk 3]
```diff
@@ -47,7 +47,7 @@ COM_WORKFLOW_MODIFIED_AT_ASC="Bearbeitungsdatum aufsteigend"
 COM_WORKFLOW_MODIFIED_AT_DESC="Bearbeitungsdatum absteigend"
 COM_WORKFLOW_MODIFIED_LABEL="Bearbeitungsdatum"
 COM_WORKFLOW_MSG_DELETE_IS_DEFAULT="Der Standard-Eintrag kann nicht gelöscht werden."
-COM_WORKFLOW_MSG_DELETE_STAGE_IS_ASSIGNED="Diese Phase wird von der Komponente verwendet."
+COM_WORKFLOW_MSG_DELETE_STAGE_IS_ASSIGNED="Diese Phase wird einem Element dieser Komponente verwendet und muss daher veröffentlicht bleiben."
```
Comment (heelc29) [inline administrator/language/de-DE/com_workflow.ini]:
```suggestion
COM_WORKFLOW_MSG_DELETE_STAGE_IS_ASSIGNED="Diese Phase wird von einem Element dieser Komponente verwendet und muss daher veröffentlicht bleiben."
```
Nur das klingt etwas _holprig_, war aber im Original PR #2673 schon so.

**Review by tecpromotion — APPROVED** [review]

**Pull request merged**